src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java

```diff
@@ -341,7 +335,7 @@ public void onFailure( @NotNull Call call, @NotNull IOException e )
                         scope.close();
                         span.end();
 
-                        logger.trace( "Failed: " + call.request().url(), e );
+                        logger.debug( "Failed: [" + method + "] " + url, e );
```

Comment:
Do we need the timestamp to match the request and response ? like this example `1700621504091.392790605444873` in content service:
` [GET /folo/track/build-A4S2BBDNVVIAA/maven/group/build-A4S2BBDNVVIAA/org/slf4j/slf4j-api/1.7.28/slf4j-api-1.7.28.jar (1700621504091.392790605444873)]`

that maybe also help to debug some issues. 

And I think it's worth to make it `INFO` level.

Reply:
The log itself has time stamp, like
`2023-11-21 16:08:55,497 DEBUG [org.com.uti.gat.ser.ProxyService_Subclass] (OkHttp http://indy.indy-stage.svc.cluster.local/...) Setting response header: Expires = 0
`
It is down to milliseconds. Do you think it is good enough? or you mean something else?

INFO is Ok.  Updated.

Reply:
Sorry for the confusing, I mean the timestamp we can identify the related request and response, if there are two same requests ( rare case ) coming, we can use this to check the related response for each.

Reply:
I see. The thing like '(1700621504091.392790605444873)' is printed by %thread. Not sure why Quarkus truncate the thread name with '...'. I will investigate it.

Reply:
I found where it is from. Updated as old indy.

Reply:
That's cool.